diff --git a/node_modules/@apollo/client/index.d.ts b/node_modules/@apollo/client/index.d.ts
index ee658f7..83bac27 100644
--- a/node_modules/@apollo/client/index.d.ts
+++ b/node_modules/@apollo/client/index.d.ts
@@ -11,17 +11,6 @@ export { DataProxy } from './cache/core/types/DataProxy';
 export { InMemoryCache, InMemoryCacheConfig, } from './cache/inmemory/inMemoryCache';
 export { defaultDataIdFromObject } from './cache/inmemory/policies';
 export * from './cache/inmemory/types';
-export { ApolloProvider } from './react/context/ApolloProvider';
-export { ApolloConsumer } from './react/context/ApolloConsumer';
-export { getApolloContext, resetApolloContext, ApolloContextValue } from './react/context/ApolloContext';
-export { useQuery } from './react/hooks/useQuery';
-export { useLazyQuery } from './react/hooks/useLazyQuery';
-export { useMutation } from './react/hooks/useMutation';
-export { useSubscription } from './react/hooks/useSubscription';
-export { useApolloClient } from './react/hooks/useApolloClient';
-export { RenderPromises } from './react/ssr/RenderPromises';
-export * from './react/types/types';
-export * from './react/parser/parser';
 export { empty } from './link/core/empty';
 export { from } from './link/core/from';
 export { split } from './link/core/split';
diff --git a/node_modules/@apollo/client/index.js b/node_modules/@apollo/client/index.js
index 78d4b9e..e30f1cb 100644
--- a/node_modules/@apollo/client/index.js
+++ b/node_modules/@apollo/client/index.js
@@ -21,16 +21,6 @@ export { ApolloCache } from './cache/core/cache.js';
 export { Cache } from './cache/core/types/Cache.js';
 export { defaultDataIdFromObject } from './cache/inmemory/policies.js';
 export { InMemoryCache } from './cache/inmemory/inMemoryCache.js';
-export { getApolloContext, resetApolloContext } from './react/context/ApolloContext.js';
-export { ApolloProvider } from './react/context/ApolloProvider.js';
-export { ApolloConsumer } from './react/context/ApolloConsumer.js';
-export { DocumentType, operationName, parser } from './react/parser/parser.js';
-export { useQuery } from './react/hooks/useQuery.js';
-export { useLazyQuery } from './react/hooks/useLazyQuery.js';
-export { useMutation } from './react/hooks/useMutation.js';
-export { useSubscription } from './react/hooks/useSubscription.js';
-export { useApolloClient } from './react/hooks/useApolloClient.js';
-export { RenderPromises } from './react/ssr/RenderPromises.js';
 export { empty } from './link/core/empty.js';
 export { from } from './link/core/from.js';
 export { split } from './link/core/split.js';
